libs/core-components/src/lib/Icon/Icon.tsx

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
 import './Icon.css';
 
+type sizes = 'xs' | 'sm' | 'md' | 'lg';
+
 type IconProps = React.PropsWithChildren<{
   className?: string;
   dataTestid?: string;
   label?: string;
   name: string;
+  size?: sizes;
```

Comment:
**Note:** Optional, because without it, icon is size of font, which is sometimes desirable.